No rule to make target 'adalib/', needed by 'adalib/nosig.o'.  Stop.

Hi again,

I'm not sure when this happened but lately when attempting to build I get the following error:

```
mkdir -p build/nosig.build
mkdir -p adainclude
ln -s /usr/lib/gcc/x86_64-pc-linux-gnu/8.1.1/adainclude/system.ads adainclude/system.ads
gcc -nostdinc -I./adainclude -gnatef -gnatg -gnatwP -gnatB -gnatVn -gnatn2 -gnatp -gnata -pipe -march=x86-64 -mtune=generic -O2 -pipe -fstack-protector-strong -fno-plt -c -o build/nosig.build/nosig.o /home/earnest/build/aur/drake-git/src/drake/source/obj/nosig.ads
make: *** No rule to make target 'adalib/', needed by 'adalib/nosig.o'.  Stop.

```

I can solve this by doing `mkdir source/adalib`.

This seems to indicate the `mkdir -p` rule for adalib isn't being run:

```make
$(RTSDIR)/adalib:
	mkdir -p $@
```

--- a/source/Makefile
+++ b/source/Makefile
@@ -421,7 +421,7 @@ all: $(EXTRA_OBJS) $(LIBS) $(ALL_ALI)
 
 tree: $(TREE)
 
-$(RTSDIR)/adalib/$(LIBGNAT): $(TREE) $(ALL_OBJS) | $(RTSDIR)/adalib/
+$(RTSDIR)/adalib/$(LIBGNAT): $(TREE) $(ALL_OBJS) | $(RTSDIR)/adalib
 	-rm $@ 2> /dev/null
 ifneq ($(ARGLIMIT),)
 	find $(LIBGNAT_BUILDDIR) -name "*.o" | xargs -L $(ARGLIMIT) $(AR) -qS $@
@@ -435,10 +435,10 @@ $(RTSDIR)/adalib/$(LDSCRIPT):
 	echo "GROUP($(LIBGNAT) AS_NEEDED($(LDFLAGS)))" > $@
 endif
 
-$(RTSDIR)/adalib/nosig.o: $(NOSIG_BUILDDIR)/nosig.o | $(RTSDIR)/adalib/
+$(RTSDIR)/adalib/nosig.o: $(NOSIG_BUILDDIR)/nosig.o | $(RTSDIR)/adalib
 	install -m 444 $(NOSIG_BUILDDIR)/nosig.o $@
 
-$(RTSDIR)/adalib/notb.o: $(NOTB_BUILDDIR)/notb.o | $(RTSDIR)/adalib/
+$(RTSDIR)/adalib/notb.o: $(NOTB_BUILDDIR)/notb.o | $(RTSDIR)/adalib
 	install -m 444 $(NOTB_BUILDDIR)/notb.o $@
 
 $(RTSDIR)/adainclude:

